changelogs/fragments/20240613_ec2_placement_group_tags.yml

```diff
@@ -1,2 +0,0 @@
-minor_changes:
```

Comment:
You should not remove this file, while the module is moved to amazon.aws collection it remains true that the module supports tags, which should be noticed in the next community.aws release